### ISSUE TITLE
numpy and pytorch predict logits result don't match

### ISSUE THREAD

**Issue by aixingxy** [kakaobrain/g2pM#16]
I made a pull requests
